components/card.scss

```diff
@@ -14,6 +14,19 @@
   word-wrap: break-word;
 }
 
+.flexCardContainer {
+  display: flex;
+}
+
+.masonryCardContainer {
+  display: inline-block;
+  margin: 10px;
+  width: 100%;
+  width: -moz-available;
+  width: -webkit-fill-available;
+  width: fill-available;
```

Comment:
`fill-available` isn't widely supported enough to be used, the fact that even chrome required the prefixed version to work indicates that. 

I played around, and I think you can re-write like so:

```
.cardMasonry {
    columns: 2 200px;
    column-gap: 10px;
    margin: 10px;
}

.masonryCardContainer {
  display: inline-block;
  margin: 5px 0;
  width: 100%;
}
```

Reply:
https://caniuse.com/#search=fill-available

`Chrome does not yet unprefix stretch (aka fill/fill-available), because the CSSWG is not ready for that yet. See Chromium bug.`

Also I betcha Safari won't like it

Reply:
Ah nice, thanks for help

Reply:
Ah nice, thanks for help